src/js/shield_defs.js

```diff
@@ -4285,6 +4288,17 @@ export function loadShields() {
     "Blue Route": bransonRouteShield(Color.shields.blue, Color.shields.white),
   };
 
+  shields["US:NH:Turnpike"].overrideByName = {
+    "Everett Turnpike": {
```

Comment:
Just double-checking: can we be sure that each individual way along this route has this as the `name`? Unfortunately, OpenMapTiles only exposes the names of individual ways rather than the route relation as a whole.

Reply:
Looks like the end of the Everett Turnpike needs to be fixed, as well as a toll booth on both turnpikes. I'll take care of that. Otherwise, there isn't a reason for the name to be changed.

Reply:
The way names are all correct now.